src/Components/SortAndFilter.js

```diff
@@ -44,7 +44,7 @@ export function SortAndFilter() {
             value={priceRangeControl}
             type="range"
             min="50"
-            max="1000"
+            max="5000"

```

Comment:
the min and max value can be calculated dynamically based on the price of the current list of items

Reply:
Thanks for pointing that out . It didnt occur to me at all !!